### PR TITLE
added vertical axis to indi guidance and cleanup

### PR DESCRIPTION
**Pull request by EwoudSmeur**
the vertical acceleration is now also controlled, which requires one parameter: the relation between thrust command and specific thrust delivered. Also, this PR brings some code clean-up and more comments.

### CODE REVIEW

File: sw/airborne/firmwares/rotorcraft/guidance/guidance_indi.c [hunk 1]
```diff
@@ -184,10 +265,18 @@ void guidance_indi_calcG(struct FloatMat33 *Gmat) {
   RMAT_ELMT(*Gmat, 2, 2) = cphi*ctheta;
 }
 
-void stabilization_attitude_set_setpoint_rp_quat_f(bool in_flight, int32_t heading)
+/**
+ * @param indi_rp_cmd roll/pitch command from indi guidance
+ * @param in_flight in flight boolean
+ * @param heading the desired heading
```
Comment (flixr) [inline sw/airborne/firmwares/rotorcraft/guidance/guidance_indi.c]: if you already document the parameters, please also specify the units....

File: sw/airborne/firmwares/rotorcraft/guidance/guidance_indi.c [hunk 2]
```diff
@@ -57,6 +62,10 @@ float roll_filtdd = 0;
 float pitch_filt = 0;
 float pitch_filtd = 0;
 float pitch_filtdd = 0;
+float T_act = 0;
```
Comment (flixr) [inline sw/airborne/firmwares/rotorcraft/guidance/guidance_indi.c]: minor note: per our conventions, variables are usually all lower case...
In any case `thrust_act` might be more readable... on capital T I first thought about time period or Tesla

**Comment by EwoudSmeur** [comment]
So this looks good now?

File: sw/airborne/firmwares/rotorcraft/guidance/guidance_indi.c [hunk 3]
```diff
@@ -184,10 +265,18 @@ void guidance_indi_calcG(struct FloatMat33 *Gmat) {
   RMAT_ELMT(*Gmat, 2, 2) = cphi*ctheta;
 }
 
-void stabilization_attitude_set_setpoint_rp_quat_f(bool in_flight, int32_t heading)
+/**
+ * @param indi_rp_cmd roll/pitch command from indi guidance [rad]
+ * @param in_flight in flight boolean
+ * @param heading the desired heading [rad]
```
Comment (flixr) [inline sw/airborne/firmwares/rotorcraft/guidance/guidance_indi.c]: nitpick: this is radians in BFP with INT32_ANGLE_FRAC, whereas the indi_rp_cmd is really in radians...

**Comment by flixr** [comment]
sorry for the delay... looks good

**Pull request merged**